ros/src/i2c_com/scripts/i2c.py

```diff
@@ -1,17 +1,48 @@
 #! /usr/bin/python
 import rospy
-from shared_msgs.msg import i2c_msg
+import time
+import smbus
+from BNO055 import BNO055
+from TYS01 import TSYS01
+from ms5837 import MS5837
+from shared_msgs.msg import imu_msg
+from shared_msgs.msg import temp_msg
+from shared_msgs.msg import depth_msg
+
+IMU_PUB_RATE = 50
+TEMP_RATIO_RATE = 4
 
 def message_received(msg):
   # This runs on a seperate thread from the pub
   pass
 
 if __name__ == "__main__":
   rospy.init_node('i2c_node')
-  pub = rospy.Publisher('i2c_rx', i2c_msg,
-      queue_size= 100)
-  sub = rospy.Subscriber('i2c_tx', i2c_msg,
-      message_received)
-
-  # TODO: I2C related activities
-  rospy.spin()
+  imu_pub = rospy.Publisher('imu_data', imu_msg, qeue_size = 100)
+  imu_pub.rate(IMU_PUB_RATE)
+  temp_pub = rospy.Publisher('temp_data', temp_msg, qeue_size = 100)
+  depth_pub = rospy.Publisher('depth_data', depth_msg, qeue_size = 100)
+  imu = BNO055()
+  temp = TYS01()
+  depth = ms5837()
+  temp.init()
```

Comment:
Kinda odd that this is the only class with an init function? Consider merging into the constructor?